app/controllers/yard_sales_controller.rb

```diff
@@ -0,0 +1,5 @@
+class YardSalesController < ApplicationController
+  def show
+    @yardsale = YardSale.find(params[:id])
```

Comment:
Looks good. I doesn't have to be addressed in this PR, but it may be a good idea to handle ids that are invalid or don't correspond with any records. 404 maybe.

Reply:
added a check if the id doest not exists. Redirects to 404